setup.py

```diff
@@ -54,7 +53,7 @@ def read(fname):
           "console_scripts": ["mbedflash=mbed_flasher:mbedflash_main",],
       },
       install_requires=[
-          "mbed-ls==1.2.16" if "win" in sys.platform else ["mbed-ls==1.2.14"],
+          "mbed-ls==1.3.5",
```

Comment:
> Brings mbed-ls >=v1.3.5 support.
Hardcode `==1.3.5` ?

Reply:
for now we have to hard code it because there is still workaround for IOTMORF-1657 and there is no API tests for https://github.com/ARMmbed/mbed-ls/pull/280. Implementation point of view it should be compatible with mbed-ls 1.x.x versions so when above issues is fixed we can create patch release with something like `>=1.3.6`